content/courses/program-optimization/rust-macros.md

```diff
@@ -826,7 +852,7 @@ pub fn instruction_builder(input: TokenStream) -> TokenStream {
         _ => panic!("The syn::Data variant is not supported: {:#?}", data),
     };
 
-    eprintln! {"{:#?}", fields};
+ 	eprintln! ("{:#?}", fields);
```

Comment:
Please double check formatting in the code. I normally just make a new `.rs`file and let `rustfmt` do it's thing, per CONTRIBUTING.